CHANGES

```diff
@@ -1,3 +1,34 @@
+Release 1.0 (development)
+----------------------------------
+
+.. note::
+
+   This version is the first official release of RAUC hawkBit Updater.
+
+.. rubric:: Enhancements
+
+* Allow following HTTP redirects
+
+.. rubric:: Testing
+
+* Switch from Travis-CI to GitHub actions.
+* Add pytest-based testsuite
+
+.. rubric:: Code
+
+* Refactoring
+
+.. rubric:: Documentation
+
+* Add sphinx-based user documentation
+
+.. rubric:: Bug fixes
+
+* Memoryleaks
+* Use-after-frees
+* Raceconditions
+* Fix free storage space calculation overflowing
+
 Release 0.1 (in development)
```

Comment:
We could skip this section as it was never really 'released'. Or you tag the initial version as 0.1, but then probably the CHANGES summary for 1.0 is quite unverbose.

Reply:
We just skip the changes as this will be the initial official release